content/v2/csidriver/features/powerscale.md

```diff
@@ -1,570 +1,659 @@
+In case of a failure, one of the standby pods becomes active and takes the position of leader. This is achieved by using native leader election mechanisms utilizing `kubernetes leases`.
+
+Additionally by leveraging `pod anti-affinity`, no two-controller pods are ever scheduled on the same node.
+
+To increase or decrease the number of controller pods, edit the following value in `myvalues.yaml` file:
+```
+controllerCount: 2
+```
+
+>**NOTE:** The default value for controllerCount is 2. It is recommended to not change this unless really required. Also, if the controller count is greater than the number of available nodes (where the pods can be scheduled), some controller pods will remain in a Pending state.
+
+If you are using the `dell-csi-operator`, adjust the following value in your Custom Resource manifest
+```
+replicas: 2  
+```
+For more details about configuring Controller HA using the Dell CSI Operator, refer to the [Dell CSI Operator documentation](../../installation/operator/#custom-resource-specification).
+
+## Ephemeral Inline Volume
+
+The CSI PowerScale driver version 1.4.0 and later supports CSI ephemeral inline volumes.
+
+This feature serves as use cases for data volumes whose content and lifecycle are tied to a pod. For example, a driver might populate a volume with dynamically created secrets that are specific to the application running in the pod. Such volumes need to be created together with a pod and can be deleted as part of pod termination (ephemeral). They get defined as part of the pod spec (inline).
+ 
+At runtime, nested inline volumes follow the lifecycle of their associated pods where the driver handles all phases of volume operations as pods are created and destroyed.
+ 
+The following is a sample manifest for creating CSI ephemeral Inline Volume in pod manifest with CSI PowerScale driver.
+
+```yaml
+kind: Pod
+apiVersion: v1
+metadata:
+  name: my-csi-app-inline-volume
+spec:
+  containers:
+    - name: my-frontend
+      image: busybox
+      command: [ "sleep", "100000" ]
+      volumeMounts:
+        - mountPath: "/data"
+          name: my-csi-volume
+  volumes:
+    - name: my-csi-volume
+      csi:
+        driver: csi-isilon.dellemc.com
+        volumeAttributes:
+          size: "2Gi"
+          ClusterName: "cluster1"
+```
+
+This manifest creates a pod in a given cluster and attaches a newly created ephemeral inline CSI volume to it.
+
+## Topology
+### Topology Support
+
+The CSI PowerScale driver version 1.4.0 and later supports Topology by default which forces volumes to be placed on worker nodes that have connectivity to the backend storage, as a result of which the nodes which have access to PowerScale Array are appropriately labeled. The driver leverages these labels to ensure that the driver components (controller, node) are spawned only on nodes wherein these labels exist. 

```

Comment:
Delete "The". Remove comma after storage and insert a period (full stop). Replace "as a result of which the " with "This results in". Replace "are appropriately"  with "being appropriately"

Reply:
Addressed the comment